src/clj/rems/application/model.clj

```diff
@@ -566,13 +453,13 @@
             enrich-members)))
 
 (defn enrich-workflow-handlers [application get-workflow]
-  (if (= :workflow/dynamic (get-in application [:application/workflow :workflow/type]))
-    (let [workflow (get-workflow (get-in application [:application/workflow :workflow/id]))
-          handlers (get-in workflow [:workflow :handlers])]
+  (let [workflow (get-workflow (get-in application [:application/workflow :workflow/id]))
+        handlers (get-in workflow [:workflow :handlers])]
+    (if handlers
       (-> application
           (assoc-in [:application/workflow :workflow.dynamic/handlers] handlers)
```

Comment:
do we want to change the name of the key? `:workflow.dynamic/handlers` sounds a bit wrong

Reply:
I guess we need some migrations, for this and maybe other naming related improvements. In the future this could be `:workflow/handlers` as a more common key even if not all workflow types would have handlers.

Reply:
Created an idea ticket for this.